### PR TITLE
Extending the columns similar to nisq analyser

### PR DESCRIPTION
**Pull request by emymaria**
Adding the columns to execution tab of implementation

### CODE REVIEW

File: src/app/components/algorithms/implementation-view/implementation-execution/implementation-execution.component.ts [hunk 1]
```diff
@@ -9,8 +9,16 @@ import { exhaustMap, first, map, switchMap, tap } from 'rxjs/operators';
...
-import { CompilerSelectionDto } from 'api-nisq/models';
+import {
+  ImplementationService,
+  QpuSelectionResultService,
+  RootService,
+} from 'api-nisq/services';
+import {
+  CompilerSelectionDto,
+  QpuSelectionJobDto,
+  QpuSelectionResultDto,
```
Comment (salmma) [inline src/app/components/algorithms/implementation-view/implementation-execution/implementation-execution.component.ts:20]: please, remove unused imports

**Review by salmma — CHANGES_REQUESTED** [review]
small comment

**Review by salmma — CHANGES_REQUESTED** [review]
implementation-execution part LG
automatic refresh-branch can be deleted?
nisq-analyzer part: execute button does not work.

**Comment by emymaria** [comment]
> implementation-execution part LG automatic refresh-branch can be deleted? nisq-analyzer part: execute button does not work.

compiler-comparision-automated-refresh can be removed

**Review by salmma — APPROVED** [review]
LG
Sorting for NISQ Analyzer in context of an algorithm (multiple tables) should be fixed
Checking if QPU data is up-to-date should be fixed.

**Pull request merged**